backend_addon/{{ cookiecutter.__folder_name }}/Makefile

```diff
@@ -45,77 +46,88 @@ all: build
 help: ## This help message
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-$(BIN_FOLDER)/pip $(BIN_FOLDER)/tox $(BIN_FOLDER)/pipx $(BIN_FOLDER)/uv $(BIN_FOLDER)/mxdev:
-	@echo "$(GREEN)==> Setup Virtual Env$(RESET)"
-	$(PYTHON) -m venv $(VENV_FOLDER)
-	$(BIN_FOLDER)/pip install -U "pip" "uv" "wheel" "pipx" "tox" "pre-commit"
-	if [ -d $(GIT_FOLDER) ]; then $(BIN_FOLDER)/pre-commit install; else echo "$(RED) Not installing pre-commit$(RESET)";fi
+requirements-mxdev.txt: ## Generate constraints file
+	@echo "$(GREEN)==> Generate constraints file$(RESET)"
+	@echo '-c https://dist.plone.org/release/$(PLONE_VERSION)/constraints.txt' > requirements.txt
+	@uvx mxdev -c mx.ini
+
+$(VENV_FOLDER): requirements-mxdev.txt ## Install dependencies
+	@echo "$(GREEN)==> Install environment$(RESET)"
+	@uv venv $(VENV_FOLDER)
+	@uv pip install -r requirements-mxdev.txt
 
-instance/etc/zope.ini: $(BIN_FOLDER)/pip  ## Create instance configuration
+.PHONY: sync
+sync: $(VENV_FOLDER) ## Sync project dependencies
+	@echo "$(GREEN)==> Sync project dependencies$(RESET)"
+	@uv pip install -r requirements-mxdev.txt
```

Comment:
In both pip-tools and uv, the sync command both installs missing packages and removes extra packages that are not in the requirements. This command only installs new packages, so I would not call it `sync`.

Reply:
@stevepiercy I know @ericof has tried it that way on several projects recently, so I assume he has reasons for not introducing that here yet. For one thing, he ran into trouble using the Plone constraints within pyproject.toml because uv doesn't support any way to refer to the constraints files on dist.plone.org.

Reply:
I just found this in uv's docs: https://docs.astral.sh/uv/concepts/projects/dependencies/#url

Reply:
Oh, nm, that's for a single package.

Reply:
Or how about request the remote constraints file, save it in a temp location, then use this:

```
uv add -r requirements.txt
```

https://docs.astral.sh/uv/concepts/projects/dependencies/#importing-dependencies

Reply:
You can do that but then it's annoying to update the constraints later. Let's wait and see what @ericof says about the decision because I'm just guessing what his reasons are.

Reply:
If you repeat the process of download remote constraints file, save it, then iterate over each line in the constraints, and call the following:

```shell
uv add "package-name>0.1.0" --upgrade-package package-name
```

...it should just work.

https://docs.astral.sh/uv/concepts/projects/dependencies/#changing-dependencies